code/style.css

```diff
@@ -1,4 +1,310 @@
-html {
-  background: #0025ff;
-  color: #fff;
+/*MOBILE (max-width=667px) */
+
+body {
+  min-width: 500px;
```

Comment:
What if I have a phone that is less than 500px wide? 🤔 
<img width="498" alt="Skärmavbild 2021-01-29 kl  15 06 48" src="https://user-images.githubusercontent.com/51529201/106284519-9e1d8e00-6243-11eb-9cfe-0f0b216966f4.png">